app/assets/javascripts/infinite-scroll.js.coffee

```diff
@@ -0,0 +1,41 @@
+$ ->
```

Comment:
This is the only `CoffeeScript` file we have in the Cypress code base. For consistency's sake, we may want to rewrite this into plain old JavaScript (sorry...)